roboteam_ai/src/Switches.cpp

```diff
@@ -118,7 +118,8 @@ std::vector<std::string> Switches::tacticJsonFileNames = {
         "normal_play_offense_tactic",
         "time_out_tactic",
         "two_robot_ballplacement_tactic",
-        "calibration_tactic"
+        "calibration_tactic",
+        "new_robot_tactic"
```

Comment:
What does the new robot tactic do? is it a test tactic? in that case prepend it with `test_`

Reply:
removed it, no need to merge this